### DISCUSSION TITLE
How to change the text of Run Failed email notification?

### DISCUSSION THREAD

**Discussion by andreamussap** [community/community#26497] [Actions]
<p>Hi all. I’ve added a GitHub Actions to the CI workflow of my project, but unfortunatelly it doesn’t work when the pull request is sent via a Fork branch (it’s seems that this is an issue with lots of GitHub Actions!).</p>
<p>This doesn’t prevent the merge of the PR, but when the user who sent the PR receives a notification like this gets concerned, the user thinks that they did something wrong.</p>
<p>Run failed for cms/amussapvordel/axway-open-docs/apigtw_devguide/test-1 (d56c844)<br>
Repository: Axway/axway-open-docs<br>
Workflow: size-label<br>
Duration: 27.0 seconds<br>
Finished: 2020-03-13 15:45:06 UTC</p>
<p>Since I still didn’t find a better Action to replayce the ‘size-label’ one, as a workaround I’d like to know how to change the message on the notification to add some information to tell the user that they don’t need to worry about it.</p>

**Comment by francisfuzz** [comment]
<p><a class="mention" href="/andreamussap">@andreamussap</a> - Hey there! It’s not possible to change the text of that email notification at this time. However, our product team is keen to hear this sort of feedback and we encourage you to share it with them here:</p>
<p><a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
<p>With respect to a workflow _not _triggering when the pull request is sent from a fork’s branch, could you share a specific instance of that with us here? I’d love to look into that and see what might be happening there. :eyes:</p>

**Comment by andreamussap** [comment]
<p><a class="mention" href="/francisfuzz">@francisfuzz</a> thanks for repling to my question. If there’s no issue/ solution you can close this ticket.</p>
<p>I’ve sent the feedback as per your suggestion.</p>
<p>As for the GitHub Actions/Fork branch issue, please have a look at this - <a href="https://github.community/t/actions-not-working-correctly-for-forks/16649">https://github.community/t5/GitHub-Actions/Actions-not-working-correctly-for-forks/td-p/35545</a></p>
<p>/Andrea</p>

**Comment by francisfuzz** [comment, marked as answer]
<p><em>&gt; If there’s no issue/ solution you can close this ticket.</em></p>
<p><em>&gt; I’ve sent the feedback as per your suggestion.</em></p>
<p><a class="mention" href="/andreamussap">@andreamussap</a> - Thanks for doing that! I’ll go ahead and mark this as the “accepted solution” given where we are. Also, thanks for sharing that topical reference around Actions and forks. I noticed one of my colleagues, <code>ethomson</code>, working on GitHub Actions replied to and acknowleged that topic. I know that other people are adding their input as well and while I can’t speak to any other new updates as of yet, we definitely recommend following our GitHub Blog’s Changelog for the latest updates:</p>
<p><a href="https://github.blog/changelog/" rel="nofollow noopener">https://github.blog/changelog/</a></p>
<p>Hope this helps!</p>
